tsl/test/expected/chunk_api.out

```diff
@@ -100,6 +100,13 @@ LIMIT 1 \gset
 SELECT slices AS "SLICES"
 FROM _timescaledb_functions.show_chunk(:'CHUNK_SCHEMA'||'.'||:'CHUNK_NAME') \gset
 TRUNCATE chunkapi;
+-- adding an existing table with additional columns should fail
+\set ON_ERROR_STOP 0
+CREATE TABLE extracol_chunk (time timestamptz NOT NULL, device int, temp float, extra_col int);
+SELECT * FROM _timescaledb_functions.create_chunk('chunkapi', :'SLICES', :'CHUNK_SCHEMA', :'CHUNK_NAME', 'extracol_chunk');
+ERROR:  table "extracol_chunk" contains column "extra_col" not found in parent "chunkapi"
+DETAIL:  The new chunk can contain only the columns present in parent.
```

Comment:
Is it possible to add columns with an ALTER after that, or is that also blocked?